bin/web.js

```diff
@@ -231,181 +230,114 @@ fs.readFile(path.join(__dirname, '../config/config.json'), 'utf8', (err, data) =
     }
 
     app.post(urls.upload, (req, res) => {
-      const { storeId } = req
-      const { bucket, host } = spaces[0]
-      logger.log(`${storeId} Uploading...`)
-      // unique object key
-      let key = '@v3/'
+      logger.log(`[storage-api] Upload start...`,)
 
+      // Validate file
       localUpload.single('file')(req, res, (err) => {
         if (err) {
-          const usrMsg = {
+          sendError(res, 400, 3001, err.message, {
             en_us: 'This file cannot be uploaded, make sure it is a valid image with up to 2mb',
             pt_br: 'O arquivo não pôde ser carregado, verifique se é uma imagem válida com até 2mb'
-          }
-          sendError(res, 400, 3001, err.message, usrMsg)
+          })
         } else {
+
+          // Retrieve request parameters
+          let key = '@v3/'
+          const { bucket, host } = spaces[0]
+
+          // Retrieve local image data
           let dir = req.query.directory
           if (typeof dir === 'string' && dir.charAt(0) === '/') {
-            // remove first char to not duplicate first bar
-            // normalize, then remove empty paths
             dir = dir.substr(1).replace(/[^\w-/]/g, '').replace('//', '')
             if (dir.length) {
               key += dir.toLowerCase() + '/'
             }
           }
-          // keep filename
+
+          // Keep filename
           const filename = req.file.originalname.replace(/[^\w-.]/g, '').toLowerCase()
-          key += `${Date.now()}-${filename}`
-          const { mimetype } = req.file
-
-          runMethod('putObject', {
-            ...baseS3Options,
-            ContentType: mimetype,
-            Key: `${storeId}/${key}`,
-            Body: req.file.buffer
-          })
-          // S3 Response
-          .then(() => {
-            logger.log(`${storeId} ${key} Uploaded to ${bucket}`)
-            // zoom uploaded
-            const mountUri = (key, baseUrl = cdnHost || host) => `https://${baseUrl}/${storeId}/${key}`
-            const uri = mountUri(key)
-            const picture = { zoom: { url: uri } }
-            const pictureBytes = {}
-            // resize/optimize image
-            let i = -1
-            let transformedImageBody = null
-
-            const respond = () => {
-              logger.log(`${storeId} ${key} ${bucket} All optimizations done`)
-              res.json({ bucket, key, uri, picture })
+          key += `${v4()}-${filename}`
```

Comment:
```suggestion
          key += `${Date.now()}-${filename}`
```

acho que o uuid só pra isso aqui é exagero, principalmente porque ainda concatena com o filename original..